src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/category/CategoriesFieldset.java

```diff
@@ -148,15 +140,15 @@ public CategoriesFieldset clickEdit() {
 
   /**
    * For mobile "All" category is not visible, that is why it's always false - can not be edited.
-   * TODO: please remember to change this line when SOC-3793 is done:
-   * TODO: "isCategoryEditable(editableCategoryAll)"
+   * TODO: please remember to change this line when SOC-3793 is done: TODO:
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)